Makefile

```diff
@@ -670,14 +676,14 @@ deb_php:
 		 --package $(BIN_PATH)/deb/$(PHP_PACKAGE_NAME)_$(NAME_SUFFIX) \
 		 --architecture $(DEBIAN_ARCHITECTURE) \
 		 --version $(VERSION)+$(OS_CODENAME) \
-		 --depends php$(PHP_VERSION_FULL) \
+		 --depends "$(PHP_DEPENDENCIES)" \
 		 --deb-priority optional \
 		 --after-install $(PHP_POST_INSTALL_SCRIPT) \
 		 --before-remove $(PHP_PRE_UNINSTALL_SCRIPT) \
 		 --category $(PACKAGE_CATEGORY) \
 		 --deb-no-default-config-files \
 		 $(PHP_LIB_MAP)
-	@find $(BIN_PATH) -name \*.
+	@find $(BIN_PATH) -name \*.deb
```

Comment:
I think it would be better to filter out other packages (`libthemis` and `libthemis-dev`) and show only `libphpthemis`, which build in the step.